tdp/cli/utils.py

```diff
@@ -217,3 +217,14 @@ def decorator(fn: FC) -> FC:
         return decorator
     else:
         return decorator(func)
+
+
+class CatchGroup(click.Group):
+    """Catch exceptions and print them to stderr."""
+
+    def __call__(self, *args, **kwargs):
+        try:
+            return self.main(*args, **kwargs)
+
+        except Exception as e:
+            print("Error:", e)
```

Comment:
Use click instead of the standard lib:

```suggestion
            click.echo(f"Error: {e}", err=True)
```

Reply:
done